charts/opentelemetry-demo/Chart.lock

```diff
@@ -1,7 +1,7 @@
 dependencies:
 - name: opentelemetry-collector
   repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-  version: 0.55.1
+  version: 0.57.2
```

Comment:
I bumped this because the `0.57.2` because that version added more attributes in the k8s attributes processor: https://github.com/open-telemetry/opentelemetry-helm-charts/pull/789

I'll send a separate PR to bump to that version and will rebase.

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/pull/821